tests/monit_test/test_monit_service_status.py

```diff
@@ -22,7 +22,5 @@ def test_monit_service_status(duthost):
     status_line = monit_service_status_info["stdout_lines"][0].strip()
     if "active" in status_line:
         logger.info("Monit service is running.")
-    elif "inactive" in status_line:
-        pytest.fail("Monit service is in 'dead' state and is not running.")
-    elif "failed" in status_line:
-        pytest.fail("Monit service is in 'failed' state and is not running.")
+    else:
+        pytest.fail("Monit is not running.")
```

Comment:
Suggest appending `status_line` to the output, might give a clue as to what's wrong.

Reply:
I appended possible reason `dead|failed` to the output. Is it appropriate?